src/dos/drive_local.cpp

```diff
@@ -207,7 +223,7 @@ bool localDrive::FindFirst(char * _dir,DOS_DTA & dta,bool fcb_findfirst) {
 		}
 	} else {
 		if (sAttr == DOS_ATTR_VOLUME) {
-			if ( strcmp(dirCache.GetLabel(), "") == 0 ) {
+			if (strcmp(dirCache.GetLabel(), "") == 0) {
```

Comment:
Perhaps `strlen` would be easier to read in this case ;)

The best solution would be for `dirCache.GetLabel()` to return a `std::string`, but I think it's out of scope for this review.

Reply:
argh.. indeed! Fixed.